utils/dgcClientSelector.ts

```diff
@@ -61,16 +62,20 @@ export const createDGCClientSelector = (
       DGC_UAT_SERVER_CA
     )
   });
-  const loadTestClient = createClient({
-    basePath: "",
-    baseUrl: DGC_LOAD_TEST_URL.href,
-    fetchApi: getFetchWithClientCertificate(
-      env,
-      DGC_LOAD_TEST_CLIENT_CERT,
-      DGC_LOAD_TEST_CLIENT_KEY,
-      DGC_LOAD_TEST_SERVER_CA
-    )
-  });
+  const loadTestClient = {
+    ...createClient({
+      basePath: "",
+      baseUrl: DGC_LOAD_TEST_URL.href,
+      fetchApi: getFetchWithClientCertificate(
+        env,
+        DGC_LOAD_TEST_CLIENT_CERT,
+        DGC_LOAD_TEST_CLIENT_KEY,
+        DGC_LOAD_TEST_SERVER_CA
+      )
+    }),
+    // Mock getCertificate request with a fake certificate
+    getCertificateByAutAndCF: mockGetCertificateByAutAndCF
+  };
```

Comment:
Is useful to create the whole client to never use it?
We could replace it with something like that.

```suggestion
  const loadTestClient = {
    // Mock getCertificate request with a fake certificate
    getCertificateByAutAndCF: mockGetCertificateByAutAndCF,
    managePreviousCertificates: mockManagePreviousCertificates
  };
```
And create the relative mock function on mockDGCClient.ts:
`
const aMockedError: r.TypeofApiResponse<ManagePreviousCertificatesT> = {
  headers: {},
  status: 500,
  value: { error: { message: "Operation not implemented for tester CF." } }
};
`
`
export const mockManagePreviousCertificates: DGCClient["managePreviousCertificates"] = async _body =>
  E.right(aMockedError);
`

Reply:
Sure, I din't want to make other changes, but it's a good suggestion. I'll change it

Reply:
or, we can make `loadTestClient` a lazy call in the form `() => something` (this would be good for other clients as well).

Reply:
In [c361232](https://github.com/pagopa/io-functions-eucovidcerts/pull/76/commits/c36123234c238f876d9296bf1cfe4270bf53b7d6) I added Fabrizio's suggestion, by creating a complete fake client into `mockDGCClient.ts` (now renamed `fakeDGCClient.ts`)

In future we can make `loadTestClient` a lazy call, if needed, with almost no effort.